@(#)PROGRAM:XCTestSwiftSupport  PROJECT:XCTest-17143.1
ffff
XCTIssue
XCUIGestureVelocity
__C_Synthesized
XCTestError
NXCTestErrorCode
Code
NXCTestErrorCode
XCTSkip
XCTest
UnwrappingOptional
XCTest
ObservedErrorContext
__swift5_proto
raise
dictionaryWithObjects:forKeys:count:
description
type
setType:
compactDescription
setCompactDescription:
detailedDescription
setDetailedDescription:
sourceCodeContext
setSourceCodeContext:
associatedError
setAssociatedError:
attachments
setAttachments:
addAttachment:
copy
mutableCopy
initWithType:compactDescription:detailedDescription:sourceCodeContext:associatedError:attachments:
initWithFilePath:lineNumber:
message
initWithExpectedValue:evaluatedExpression:message:filePath:lineNumber:
initWithError:evaluatedExpression:message:filePath:lineNumber:
initWithException:evaluatedExpression:message:filePath:lineNumber:
initWithExplanation:evaluatedExpression:message:filePath:lineNumber:
reason
_installErrorObserver
_observeErrors
currentThread
threadDictionary
__swift_setObject:forKeyedSubscript:
initWithCallStackAddresses:location:
__swift_objectForKeyedSubscript:
callStackReturnAddresses
type
objc
exception
unknown
value
/Library/Caches/com.apple.xbs/Sources/XCTest_Sim/XCTest-17143.1/Sources/libXCTestSwiftSupport/XCTContext_SwiftExtensions.swift
Fatal error
XCTContext.runActivity(named:block:) failed because activities are disallowed in the current configuration.
XCTAssertNil failed: threw error "
XCTAssertNotNil failed: threw error "
XCTUnwrap failed: threw error "
XCTAssertTrue failed: threw error "
XCTAssertFalse failed: threw error "
XCTAssertEqual failed: threw error "
XCTAssertNotEqual failed: threw error "
XCTAssertGreaterThan failed: threw error "
XCTAssertGreaterThanOrEqual failed: threw error "
XCTAssertLessThan failed: threw error "
XCTAssertLessThanOrEqual failed: threw error "
XCTAssertThrowsError failed: throwing an unknown exception
XCTAssertThrowsError failed: throwing 
XCTAssertThrowsError failed: threw error "
XCTAssertThrowsError failed: did not throw an error
XCTAssertNoThrow failed: threw error "
v8@?0
com.apple.dt.xctest.swift-error-observation.enabled
com.apple.dt.xctest.swift-error-observation.last-observed-error
B16@0:8
@?16@0:8
@"XCTIssue"8@?0
failed: caught error: "
_swift_willThrow
__TEXT
swift_getObjCClassMetadata
_TtCs12_SwiftObject
swift_getTypeContextDescriptor
swift_getExistentialTypeMetadata
objc_addLoadImageFunc
__swift5_proto
ySo8XCTIssueCG
So8XCTIssueC
So7NSErrorC
So8NSNumberC
$ss21_ObjectiveCBridgeableP
$s10Foundation21_BridgedStoredNSErrorP
$sSY
$s10Foundation18_ErrorCodeProtocolP
ySS_yptG
SS_ypt
So21XCTSkippedTestContextC
ySSypG
ypSg
SDySSypG
_pIgzo_
_pSg
So8NSThreadC
So7NSArrayC
$ss27ExpressibleByIntegerLiteralP
$ss25ExpressibleByFloatLiteralP
underlyingIssue
value
_nsError
rawValue
_ObjectiveCType
Code
RawValue
_ErrorType
skippedTestContext
errorCode
errorCode
errorUserInfo
error
callStackReturnAddresses
IntegerLiteralType
FloatLiteralType
